roles/bibdata/tasks/mounts.yml

```diff
@@ -38,6 +38,23 @@
     state: mounted
   when: running_on_server
 
+- name: Create data directory
+  file:
+    path: '{{ bibdata_data_dir | dirname }}'
+    state: directory
+    recurse: yes
+    owner: '{{ deploy_user }}'
+    group: '{{ deploy_user }}'
+
+- name: Make sure the mount directory exists
+  file:
+    path: '/mnt/bibdata_share/{{ bibdata_mount_env }}'
+    state: directory
+    recurse: no
+    owner: '{{ deploy_user }}'
+    group: '{{ deploy_user }}'
+  run_once: true
```

Comment:
With this line added, if you run the role on multiple machines (for example, if we spin up 3 VMs to run bibdata), this task will only run on the first machine and not on the others

Is that the intended outcome? Why wouldn't we want to ensure that a mount directory exists on each machine?